cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -85,8 +85,6 @@ func (c *CStorPoolController) syncHandler(key string, operation common.QueueOper
 	if err != nil {
 		c.recorder.Event(cspObject, corev1.EventTypeWarning, string(common.FailedSynced), string(common.MessageResourceSyncFailure)+err.Error())
 		return err
-	} else {
-		c.recorder.Event(cspObject, corev1.EventTypeNormal, string(common.SuccessSynced), string(common.MessageResourceSyncSuccess))
```

Comment:
don't we want to record success sync event ?. or we want to ignore them ?

Reply:
recording sync event is also adding log. And, if we really want to record success event for 'sync', we need to follow 'counter' mechanism.

Reply:
I would say - we need to revisit our events usage